packages/xdl/src/credentials/Credentials.ts

```diff
@@ -1,5 +1,5 @@
 import { Platform, getConfig } from '@expo/config';
-import { ApiV2 } from '../xdl';
+import { AndroidCredentials, ApiV2 } from '../xdl';
```

Comment:
there is a import cycle here, you shoudl import this from respecitve files `../ApiV2` and `./AndroidCredentials`

Reply:
great point